src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -367,6 +370,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             JsonUtils::GetValueForKey(json, ColorSchemeKey, args->_ColorScheme);
             JsonUtils::GetValueForKey(json, ElevateKey, args->_Elevate);
             JsonUtils::GetValueForKey(json, ReloadEnvironmentVariablesKey, args->_ReloadEnvironmentVariables);
+            JsonUtils::GetValueForKey(json, KeepOpenKey, args->_KeepOpen);

```

Comment:
Is it correct that this json code and the one in line 395 is used to read form/write to a WT arguments json file? Do I need to update a json schema? (I can't find one.)

Reply:
Yep, that's pretty much it. The schema can be found at [`terminal/doc/cascadia
/profiles.schema.json`](https://github.com/microsoft/terminal/blob/main/doc/cascadia/profiles.schema.json)